runners/flink/flink_runner.gradle

```diff
@@ -285,8 +285,6 @@ createJavaExamplesArchetypeValidationTask(type: 'Quickstart', runner: 'FlinkLoca
 
 task examplesIntegrationTest(type: Test) {
   group = "Verification"
-  // Disable gradle cache
-  outputs.upToDateWhen { false }
```

Comment:
Why is this removed?

Reply:
I've used some of the configs to launch tests using FlinkRunner from the ValidatesRunner task, I'm not sure if for this case is it worth it to reuse previous outputs from the cache or not. I lean more towards the second.